.github/workflows/code-checks.yml

```diff
@@ -114,6 +114,9 @@ jobs:
       with:
         fetch-depth: 0
 
+    - name: Uncomment pytorch dependency
+      run: sed -i 's/# - pytorch/- pytorch/g' environment.yml
```

Comment:
I don't it does for this job because there's

https://github.com/pandas-dev/pandas/blob/028613021df9ecc83a546f83fb8847199c63cc3a/.github/workflows/code-checks.yml#L131

In the CI run of the commit which doesn't have this, the torch ASV check is skipped: https://github.com/pandas-dev/pandas/actions/runs/3507106640/jobs/5874585410

```
2022-11-20T08:22:57.2883873Z [  7.85%] ··· frame_ctor.From3rdParty.time_from_torch                        n/a
```